toolbox/fdc3-explained/index.html

```diff
@@ -0,0 +1,129 @@
+<html>
+
+<head>
+	<title>
+		FDC3 Explained v.2
+	</title>
+
+
+
```

Comment:
When run locally was picking up the wrong character encoding:
![image](https://user-images.githubusercontent.com/1701764/107363078-8eb70400-6ad1-11eb-9dc6-25c3541072b4.png)
Explicit encoding set resolves that
```suggestion
<meta charset="UTF-8">
```